### ISSUE TITLE
Add to Comfy Manager

### ISSUE THREAD

**Issue by drmbt** [BadCafeCode/execution-inversion-demo-comfyui#13]
It would be a lot easier to manage workflows that use these nodes if they were submitted to Comfy Manager. pretty please 🙏

**Comment by niko2020** [comment]
Yes, that would be great, and I don't understand why this code isn't part of the official comfy distribution, given that it's a very important element of the latest update. In fact, the title should be stripped of the "demo" heading.

**Comment by PrometheusDante** [comment]
Absolutely agree. Having users install it by switching to the dev-channel first to have it show up on the node pack list is nothing but inconvenient. The amount of people that struggled getting my workflows to run just because of that alone is ridiculous.

Wish the comfy team would merge these nodes into their core. I find use for these nodes in just about every workflow, but hesitate to risk getting even more people frustrated with failing to install it correctly.
